src/TournamentManager.ts

```diff
@@ -485,6 +486,29 @@ export class TournamentManager implements TournamentInterface {
 				const winnerScore = weWon ? scorePlayer : scoreOpp;
 				const loserScore = weWon ? scoreOpp : scorePlayer;
 				await this.website.submitScore(tournamentId, winner, winnerScore, loserScore);
+				// send DM to opponent
+				const opponent = score.playerDiscord;
+				await this.discord.sendDirectMessage(
+					opponent,
+					`Your opponent has successfully confirmed your score of ${scoreOpp}-${scorePlayer} for Tournament ${tournament.name}, so the score has been saved. Thank you.`
+				);
+				// report confirmation to hosts
+				const channels = tournament.privateChannels;
+				await Promise.all(
+					channels.map(async c => {
+						this.discord.sendMessage(
+							c,
+							`${mention} (${this.discord.getUsername(playerId)}) and ${this.discord.mentionUser(
+								opponent
+							)} (${this.discord.getUsername(
+								opponent
+							)}) have reported their score of ${scorePlayer}-${scoreOpp} for Tournament ${
+								tournament.name
+							} (${tournamentId}).`
```

Comment:
Prettier ironically makes this look really ugly. Maybe some variable declarations are in order?

Reply:
That's true of mass use of template strings throughout, but I don't disagree. I can take a look at cleaning them up once I've worked on this next card.